File: database/check_columns.py
#!/usr/bin/env python3
import psycopg2
from dotenv import load_dotenv
import os

load_dotenv()
conn = psycopg2.connect(os.getenv('SUPABASE_CONNECTION_STRING'))
cur = conn.cursor()

print("STORES TABLE COLUMNS:")
cur.execute("SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'stores' AND table_schema = 'public' ORDER BY ordinal_position;")
columns = cur.fetchall()
for col in columns:
    print(f"  {col[0]}: {col[1]}")

print("\nCOMPANIES TABLE COLUMNS:")
cur.execute("SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'companies' AND table_schema = 'public' ORDER BY ordinal_position;")
columns = cur.fetchall()
for col in columns:
    print(f"  {col[0]}: {col[1]}")

cur.close()
conn.close()

File: database/check-columns.js
const { Client } = require('pg');

async function checkColumns() {
  const client = new Client({
    connectionString: process.env.SUPABASE_CONNECTION_STRING,
    ssl: { rejectUnauthorized: false }
  });

  try {
    await client.connect();
    
    const tables = ['users', 'tasks', 'tables', 'orders', 'products', 'inventory_items'];
    
    for (const table of tables) {
      console.log(`\n📊 Table: ${table}`);
      console.log('═══════════════════════════════');
      
      const result = await client.query(`
        SELECT column_name
        FROM information_schema.columns
        WHERE table_schema = 'public' 
        AND table_name = $1
        AND column_name IN ('store_id', 'branch_id', 'company_id')
      `, [table]);
      
      if (result.rows.length > 0) {
        result.rows.forEach(row => console.log('  ✅', row.column_name));
      } else {
        console.log('  ❌ No store_id/branch_id/company_id');
      }
    }
    
  } finally {
    await client.end();
  }
}

checkColumns();
